Plugin/Resolver/CreateEmptyCartWithFastOrderId.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Fast\Checkout\Plugin\Resolver;
+
+use Closure;
+use \Magento\Framework\App\Http\Context;
+use Magento\QuoteGraphQl\Model\Resolver\CreateEmptyCart\Interceptor;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+
+/**
+ * Class CreateEmptyCartWithFastOrderId
+ */
+class CreateEmptyCartWithFastOrderId
+{
+    const CONTEXT_FAST_ORDER_ID_KEY = 'fast_order_id';
+
+    protected $httpContext;
+
+    public function __construct(
+        Context $httpContext
+    ) {
+        $this->httpContext = $httpContext;
+    }
+
+    public function aroundResolve(
```

Comment:
I'd recommend using beforeResolve method here

```
/**
     * @param Interceptor $subject
     * @param Field $field
     * @param $context
     * @param ResolveInfo $info
     * @param array|null $value
     * @param array|null $args
     * @return null
     */
    public function beforeResolve(Interceptor $subject, Field $field, $context, ResolveInfo $info, array $value = null, array $args = null)
    {
        if (isset($args['input']['fast_order_id'])) {
            // pass along the fast_order_id, so Plugin/Api/CartManagement can pick it up
            $this->httpContext->setValue(self::CONTEXT_FAST_ORDER_ID_KEY, $args['input']['fast_order_id'], null);
        }

        //since we're not changing any parameters
        return null;
    }